stable_baselines3/common/env_checker.py

```diff
@@ -8,12 +8,12 @@
 from stable_baselines3.common.vec_env import DummyVecEnv, VecCheckNan
 
 
-def _enforce_array_obs(observation_space: spaces.Space) -> bool:
+def _is_numpy_array_space(space: spaces.Space) -> bool:
     """
-    Whether to check that the returned observation is a numpy array
-    it is not mandatory for `Dict` and `Tuple` spaces.
+    Checks whether the space consists of a numpy array
```

Comment:
Bit of typo-ing here? This docstring could be just "Returns False if provided space is not representable as a single numpy array (e.g. `Dict` and `Tuple` spaces return False).